lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_build_prechecks.rb

```diff
@@ -9,9 +9,14 @@ def self.run(params)
 
         Fastlane::Helper::GitHelper.ensure_on_branch!('release') unless other_action.is_ci
 
+        project_root_folder = params[:project_root_folder]
+        project_name = params[:project_name]
+        build_gradle_path = params[:build_gradle_path] || File.join(project_root_folder || '.', project_name, 'build.gradle')
+        version_properties_path = params[:version_properties_path] || File.join(project_root_folder || '.', 'version.properties')
+
         message = ''
-        beta_version = Fastlane::Helper::Android::VersionHelper.get_release_version unless !params[:beta] && !params[:final]
-        alpha_version = Fastlane::Helper::Android::VersionHelper.get_alpha_version if params[:alpha]
+        beta_version = Fastlane::Helper::Android::VersionHelper.get_release_version(build_gradle_path: build_gradle_path, version_properties_path: version_properties_path) unless !params[:beta] && !params[:final]
```

Comment:
Not sure how Rubocop feels about it, but should these params each have their own line?

Reply:
Fixed in https://github.com/wordpress-mobile/release-toolkit/commit/0277441e789efbb230d6b6d971b1d07e1548ae1a